### ISSUE TITLE
Object oriented code not highlighted

### ISSUE THREAD

**Issue by iandol** [textmate/matlab.tmbundle#6]
The new classdef format for OOP code is not properly recognised for syntax highlighting. There is a pull request from 2013 which hasn't been merged to help fix this.

**Comment by IvanUkhov** [comment]
Not only TextMate suffers from it, but GitHub as well via [Linguist](https://github.com/github/linguist). I’d appreciate if somebody–I suppose @infininight is the one who is in charge–could clarify why pull request #4 cannot be merged. Thanks!

Regards,
Ivan

**Comment by infininight** [comment]
This has been corrected in a [series of commits](https://github.com/textmate/matlab.tmbundle/compare/2b0aede8c984d1a901f8f77258be870beea3045e...11e4df51c97411410611446d8e6495b1d5295a56). The grammar had not seen any attention for a very long time so I spent the weekend giving it a bit of an overhaul. There are many areas where the basically non-exist spec if very vague but I did not see any issues in the many matlab files I tested, report back if you see any remaining.

This only applies to Matlab and not the Octave grammar, it needs a similar cleanup but that has not been done. I'm still considering if the two languages have diverged enough to necessitate two separate bundles at this point.

Note: Github will take 2-4 weeks to pickup on these changes as they deploy updates in batches to the linguist project.
